flow/project.py

```diff
@@ -2316,7 +2426,7 @@ def _fetch_status(
 
         Parameters
         ----------
-        aggregates : list
+        aggregates : sequence of aggregates
```

Comment:
```suggestion
        aggregates : Sequence[aggregates]
```

Reply:
We're wildly inconsistent with docstring parameter type syntax. I honestly don't know what the right syntax is supposed to be. Would it be something like `Sequence[tuple[signac.contrib.job.Job]]`? The other problem is that it won't link classes in the Sphinx docs unless we use `:class:` syntax...

Reply:
Yes, I think that's probably appropriate, but I agree that we're not consistent and I don't want to start trying to figure that out as part of this PR. Just disregard this suggestion, and we'll try and come up with something better at a later date, perhaps as part of #404.

numpydoc is kind of inconsistent about it, but in some contexts like parameter lists it does infer the type without the needed for an explicit cross-reference. Once you write something like "sequence of aggregates" all bets are off though, I'm not sure when that'll work and when not.